node-admin/pom.xml

```diff
@@ -25,7 +25,12 @@
             <scope>provided</scope>
         </dependency>
         <dependency>
-        <groupId>com.yahoo.vespa</groupId>
+            <groupId>com.yahoo.vespa</groupId>
+            <artifactId>config-provisioning</artifactId>
+            <version>${project.version}</version>
```

Comment:
Add scope.

It would be nice to have this provided: The host admin depends on config-provision compile-scoped. We could make that provided too, and have the node admin ship it in components/.

Reply:
Its' made provided now